help.js

```diff
@@ -0,0 +1,79 @@
+var SourceDest = {
+  source: {
+    type: 'FeedId',
+    description: 'the feed which posted the contact message'
+  },
+  dest: {
+    type: 'FeedId',
+    description: 'the feed the contact message pointed at'
+  }
+}
+
+var HopsOpts = {
+  start: {
+    type: 'FeedId',
+    description: 'feed at which to start traversing graph from, default to your own feed id'
+  },
+  max: {
+    type: 'number',
+    description: 'include feeds less than or equal to this number of hops',
+  }
+}
+
+var StreamOpts = Object.assign(
+  HopsOpts, {
```

Comment:
won't this overwrite `HopsOpts`?

did you mean to do `Object.assign({}, HopsOpts, { ... })`?